tests_v2/cmux.py

```
@@ -864,6 +864,38 @@ def list_notifications(self) -> list[dict]:
+        entries = res.get("workspaces") or []
+        if workspace is None:
+            out: list[str] = []
+            for entry in entries:
+                out.extend(entry.get("surface_ids") or [])
+            return sorted(out)
+        target = self._resolve_workspace_id(workspace) if isinstance(workspace, (str, int)) else None
+        if not target:
+            return []
+        for entry in entries:
+            if entry.get("workspace_id") == target:
+                return sorted(list(entry.get("surface_ids") or []))
```

Ruff (0.15.12)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)